### PR TITLE
Fix the doc about setProgressiveSourceLoading()

### PR DESCRIPTION
**Pull request by bernie-laberge**
### Linked issues
na

### Summarize your change.

The Open RV documentation incorrectly stated that the following command could be executed to enable progressive source loading in Open RV:

`setProgressiveSourceLoading = true`

This was incorrect and fixed in this commit.
Here is the fix marked down preview:

setProgressiveSourceLoading_cmd_overview

### Describe the reason for the change.

Incorrect documentation

### If possible, provide screenshots.

### CODE REVIEW

**Review by cedrik-fuoco-adsk — APPROVED** [review]

**Review by eloisebrosseau — APPROVED** [review]

File: docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md [hunk 1]
```diff
@@ -77,9 +77,13 @@ If you need to, RV can load sources asynchronously, also known as progressive so
 
 **Use one of these methods to enable asynchronous source loading:**
 
-- Set `setProgressiveSourceLoading = true`
 - With the command line argument: `-progressiveSourceLoading 1`
 - With the environment variable: `RV_PROGRESSIVE_SOURCE_LOADING`
+- Executing the following command in an RV Python package:
```
Comment (mchesnay) [inline docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md:82]:
```suggestion
- Execute the following command in an RV Python package:
```

File: docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md [hunk 2]
```diff
@@ -77,9 +77,13 @@ If you need to, RV can load sources asynchronously, also known as progressive so
...
+```
```
Comment (mchesnay) [inline docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md:83]:
```suggestion
```python
```

**Review by mchesnay — APPROVED** [review]
A couple of syntax suggestions

File: docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md (hunk 2 above)
Comment (bernie-laberge) [reply docs/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-two.md:83]: Much nicer with it! Thanks !

**Pull request merged**